src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Classes/IWindowsSetting.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Enumerations;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes
+{
+    internal interface IWindowsSetting
```

Comment:
Do we need this interface?  This is the only class we have it on

Reply:
I don't think so. Btw. it is identically to the class itself as I can see.

Reply:
removed